test/selectors/channels.test.js

```diff
@@ -15,35 +15,49 @@ describe('Selectors.Channels', () => {
 
     const channel1 = TestHelper.fakeChannelWithId(team1.id);
     channel1.display_name = 'Channel Name';
+    channel1.last_post_at = (new Date()).getTime();
```

Comment:
Minor nit, this could also be `Date.now()` if you wanted to simplify it slightly

Reply:
Cool, thanks I'll make that change